src/scripts/fetch_user_handles.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+set -eo pipefail
+CODA_CIRCLECI_USER_NAME_COL="c-6ni4kHGNwE"
```

Comment:
Why are these hardcoded?   They are set as ENV vars in `CODA_CIRCLECI_USER_NAME_COL`,  and `CODA_CIRCLECI_USER_ALIAS_COL`, and `CODA_USER_ROSTER_TABLE_URL`?

Could this not be written as:
```sh
USER_EMAIL=""
CIRCLE_USERNAME=$CIRCLE_USERNAME
function run_main() {
    USER_ALIAS=$(curl -s -H "Authorization: Bearer ${CODA_API_TOKEN}" \
    -G --data-urlencode "query=${CODA_CIRCLECI_USER_NAME_COL}:\"${CIRCLE_USERNAME}\"" \
    ${CODA_USER_ROSTER_TABLE_URL} \
```